src/main/java/org/moss/discord/commands/moderation/PruneCommand.java

```diff
@@ -1,20 +1,18 @@
 package org.moss.discord.commands.moderation;
 
-import org.javacord.api.DiscordApi;
 import org.javacord.api.entity.channel.TextChannel;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import de.btobastian.sdcf4j.Command;
 import de.btobastian.sdcf4j.CommandExecutor;
+import org.javacord.api.entity.message.MessageSet;
+import org.javacord.api.util.logging.ExceptionLogger;
 
 public class PruneCommand implements CommandExecutor {
 
-    private static final Logger logger = LoggerFactory.getLogger(PruneCommand.class);
-
     @Command(aliases = {"!prune"}, usage = "!prune <amount>", description = "Prunes a certain amount of messages (between 2 and 100)")
-    public void onCommand(DiscordApi api, TextChannel channel, String[] args) {
-        // some code will go here I guess
+    public void onCommand(TextChannel channel, Integer[] args) {
+        channel.getMessages(100).thenCompose(MessageSet::deleteAll).exceptionally(ExceptionLogger.get());
```

Comment:
This doesn't acknowledge the amount that the command is given when run.

Reply:
I know, this is test code for now, since there is some bug that prevents the code from working correctly